src/main/java/seedu/address/ui/SuggestionTemplates.java

```diff
@@ -433,7 +433,7 @@ public static final String getString(TextFlow template) {
                 .stream()
                 .map(textElement -> (Text) textElement)
                 .map(t -> t.getText())
-                .filter(s -> !s.matches("\\[[A-Za-z{}_ ]+\\] |[A-Z_{} ]+|\\([A-Za-z_ ]+\\)"))
+                .filter(s -> !s.matches("\\[[A-Za-z{}_ ]+\\] |[A-Z_{} ]+|\\([A-Za-z_. ]+\\)"))
```

Comment:
Seems like this is repeated from the previous PR

Reply:
yeap becuase I have yet to merge the previous pr in:)